utils/cautils/offline.go

```diff
@@ -29,8 +29,17 @@ type OfflineCA struct {
 	configFile string
 }
 
+// offlineInstance is a singleton used for OfflineCA. The use of a singleton is
+// necessary to avoid double initialization. Double initializations are
+// sometimes not possible due to locks likek the on in badgerDb.
```

Comment:
Double initializations are sometimes not possible due to locks - as seen in badgerDB.